packages/benchmark/src/benchmarkerator.js

```diff
@@ -425,43 +439,96 @@ const printBenchmarkStats = stats => {
     // prettier-ignore
     `${stats.rounds} rounds in ${stats.elapsedTime}ns (${stats.timePerRound.toFixed(3)}/round})`,
   );
-  log(`${h1} ${h2} ${h3}`);
-  log(`${d1} ${d2} ${d3}`);
 
-  const data = stats.data;
-  for (const [key, entry] of Object.entries(data)) {
-    const col1 = `${key.padEnd(w1)}`;
-    const col2 = `${String(entry.delta).padStart(w2)}`;
-    const col3 = `${pn(entry.deltaPerRound).padStart(w3)}`;
+  const wc1 = 32;
```

Comment:
please make these names more intelligible, or at least a comment about the abbreviations

Reply:
Terseness was sort of the point here for layout reasons.  I've added a comment.